hack/logging-dump.sh

```diff
@@ -194,17 +195,34 @@ check_collector() {
   done
 }
 
+check_cluster-logging-operator() {
+  echo "Checking cluster-logging-operator"
+  mkdir $clo_folder
+  namespace="${NAMESPACE}"
+  pods=$(oc -n $namespace get pods -l name=cluster-logging-operator -o jsonpath={.items[*].metadata.name})
+  for pod in $pods
+  do
+    get_env $pod $clo_folder $namespace "Dockerfile-*operator*"
+    get_pod_logs $pod $clo_folder $namespace
+  done
+  oc -n $namespace get deployment cluster-logging-operator -o yaml > $clo_folder/deployment
+  csv_name="$(oc -n $namespace get csv -o name | grep 'cluster-logging-operator')"
+  oc -n $namespace get "${csv_name}" -o yaml > "${clo_folder}/csv"
+}
```

Comment:
This is really really cool! Can we expand this on the catalog and olm operator logs? Maybe also get the describe of the subscription and the install plan too. They are super useful if the CRDs are broken or similar bugs raise up.

Reply:
whoops, this file shouldn't be part of this pr!